server/base/src/main/java/org/apache/accumulo/server/util/Admin.java

```diff
@@ -506,7 +507,7 @@ private void printNameSpaceConfiguration(AccumuloClient accumuloClient, String n
       File outputDirectory)
       throws IOException, AccumuloException, AccumuloSecurityException, NamespaceNotFoundException {
     File namespaceScript = new File(outputDirectory, namespace + NS_FILE_SUFFIX);
-    FileWriter nsWriter = new FileWriter(namespaceScript);
+    BufferedWriter nsWriter = new BufferedWriter(new FileWriter(namespaceScript));
```

Comment:
This should be done in a try-with-resources block.

Reply:
Hi @ctubbsii  , thanks for your reply. I additionally move the BufferedWriter into the resource block of try. Please check. Thanks.